internal/kaniko/execute.go

```diff
@@ -224,6 +224,11 @@ func (k *Config) cmdBuilder(digestFile string) (*exec.Cmd, error) {
 		cmdArgs = append(cmdArgs, "--skip-default-registry-fallback")
 	}
 
+	if k.Target != "" {
+		fmt.Printf("Test:%v", k.Target)
```

Comment:
Dont add test in logs, Print the proper message

Reply:
Included proper message instead of Test.